### PR TITLE
unhold the binaries on the worker nodes during upgrade

### PR DESCRIPTION
**Pull request by RA489**
Steps missing to unhold the binaries on the worker nodes during upgrade

Refs https://github.com/kubernetes/website/issues/14593

### CODE REVIEW

**Comment by RA489** [comment]
@neolit123 Thanks! let me know if i can help with upgrade documents in  dev-1.15

**Comment by neolit123** [comment]
> let me know if i can help with upgrade documents in dev-1.15

no problem, i'm working on those.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12.md [hunk 1]
```diff
@@ -268,6 +270,7 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
     kubectl uncordon $NODE
     ```
 
+
```
Comment (cody-clark) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12.md]: Is this extra line intentional?

**Pull request merged**